tracer/test/Datadog.Trace.TestHelpers/FluentAssertionsExtensions/Json/FluentAssertions.Json.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why this new project? There is a FluentAssertions.Json package. Did you need to modify it? The files within this project are the original from the package? In that case, are we allowed to add the datadog copyright header to them?

Reply:
That file slipped by. I'm removing it. This is a vendoring of the JSON extensions for the FluentAssertions. I did it this way because we have Newtonsoft vendored and I didn't want to add an extra reference to the Newtonsoft library.